web/ui_utils.js

```diff
@@ -155,7 +155,7 @@ function scrollIntoView(element, spot, scrollMatches = false) {
  * Helper function to start monitoring the scroll event and converting them into
  * PDF.js friendly one: with scroll debounce and scroll direction.
  */
-function watchScroll(viewAreaElement, callback) {
+function watchScroll(viewAreaElement, callback, abortSignal = null) {
```

Comment:
It seems that passing `null` as a signal-parameter, in the `addEventListener`-call, breaks things so we probably want the following instead.
```suggestion
function watchScroll(viewAreaElement, callback, abortSignal = undefined) {
```